[2013-08-19 17:19:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:19:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:19:31] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:23:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:23:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:23:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:24:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:24:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:24:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:24:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 17:24:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:40:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:41:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:48:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:49:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:52:32] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Http\RedirectResponse::withInput() must be of the type array, string given, called in D:\Dev\workspace\epikureos\app\controllers\HomeController.php on line 36 and defined' in D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Http\RedirectResponse.php:64
Stack trace:
#0 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Http\RedirectResponse.php(64): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\Dev\workspac...', 64, Array)
#1 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(36): Illuminate\Http\RedirectResponse->withInput('lugar', 'hlhk')
#2 [internal function]: HomeController->post_busqueda()
#3 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('post_busqueda', Array)
#5 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4768): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'post_busqueda', Array)
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7838): call_user_func_array(Object(Closure), Array)
#8 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7825): Illuminate\Routing\Route->callCallable()
#9 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4779): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-08-19 18:53:22] log.ERROR: exception 'InvalidArgumentException' with message 'The HTTP status code "0" is not valid.' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:9200
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(9077): Symfony\Component\HttpFoundation\Response->setStatusCode('hjk')
#1 D:\Dev\workspace\epikureos\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\RedirectResponse.php(44): Symfony\Component\HttpFoundation\Response->__construct('', 'hjk', Array)
#2 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Redirector.php(175): Symfony\Component\HttpFoundation\RedirectResponse->__construct('http://localhos...', 'hjk', Array)
#3 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Redirector.php(117): Illuminate\Routing\Redirector->createRedirect('http://localhos...', 'hjk', Array)
#4 D:\Dev\workspace\epikureos\bootstrap\compiled.php(3338): Illuminate\Routing\Redirector->to('busqueda', 'hjk')
#5 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(36): Illuminate\Support\Facades\Facade::__callStatic('to', Array)
#6 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(36): Illuminate\Support\Facades\Redirect::to('busqueda', 'hjk')
#7 [internal function]: HomeController->post_busqueda()
#8 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#9 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('post_busqueda', Array)
#10 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4768): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'post_busqueda', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7838): call_user_func_array(Object(Closure), Array)
#13 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7825): Illuminate\Routing\Route->callCallable()
#14 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4779): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-08-19 18:54:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$lugar' (T_VARIABLE)' in D:\Dev\workspace\epikureos\app\controllers\HomeController.php:36
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-19 18:54:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$lugar' (T_VARIABLE)' in D:\Dev\workspace\epikureos\app\controllers\HomeController.php:36
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-19 18:54:41] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$lugar' (T_VARIABLE)' in D:\Dev\workspace\epikureos\app\controllers\HomeController.php:36
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-19 18:55:01] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$lugar' (T_VARIABLE)' in D:\Dev\workspace\epikureos\app\controllers\HomeController.php:36
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-19 18:55:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:55:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:55:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4945
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 18:58:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 19:05:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-08-19 19:08:01] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\Dev\workspace\epikureos\app\controllers\HomeController.php:38
Stack trace:
#0 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(38): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\Dev\workspac...', 38, Array)
#1 [internal function]: HomeController->post_busqueda()
#2 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#3 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('post_busqueda', Array)
#4 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4768): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'post_busqueda', Array)
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7838): call_user_func_array(Object(Closure), Array)
#7 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7825): Illuminate\Routing\Route->callCallable()
#8 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4779): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2013-08-19 19:09:15] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "/busqueda" as such route does not exist.' in D:\Dev\workspace\epikureos\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php:134
Stack trace:
#0 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('/busqueda', Array, true)
#1 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Redirector.php(144): Illuminate\Routing\UrlGenerator->route('/busqueda', Array)
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(3336): Illuminate\Routing\Redirector->route('/busqueda')
#3 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(38): Illuminate\Support\Facades\Facade::__callStatic('route', Array)
#4 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(38): Illuminate\Support\Facades\Redirect::route('/busqueda')
#5 [internal function]: HomeController->post_busqueda()
#6 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#7 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('post_busqueda', Array)
#8 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4768): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'post_busqueda', Array)
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#10 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7838): call_user_func_array(Object(Closure), Array)
#11 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7825): Illuminate\Routing\Route->callCallable()
#12 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4779): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#16 {main} [] []
[2013-08-19 19:12:11] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "/busqueda" as such route does not exist.' in D:\Dev\workspace\epikureos\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php:134
Stack trace:
#0 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('/busqueda', Array, true)
#1 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Redirector.php(144): Illuminate\Routing\UrlGenerator->route('/busqueda', Array)
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(3336): Illuminate\Routing\Redirector->route('/busqueda')
#3 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(38): Illuminate\Support\Facades\Facade::__callStatic('route', Array)
#4 D:\Dev\workspace\epikureos\app\controllers\HomeController.php(38): Illuminate\Support\Facades\Redirect::route('/busqueda')
#5 [internal function]: HomeController->post_busqueda()
#6 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#7 D:\Dev\workspace\epikureos\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('post_busqueda', Array)
#8 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4768): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'post_busqueda', Array)
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#10 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7838): call_user_func_array(Object(Closure), Array)
#11 D:\Dev\workspace\epikureos\bootstrap\compiled.php(7825): Illuminate\Routing\Route->callCallable()
#12 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4779): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#16 {main} [] []
[2013-08-19 19:12:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in D:\Dev\workspace\epikureos\bootstrap\compiled.php:4948
Stack trace:
#0 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4790): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException))
#1 D:\Dev\workspace\epikureos\bootstrap\compiled.php(4778): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\Dev\workspace\epikureos\bootstrap\compiled.php(483): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\Dev\workspace\epikureos\bootstrap\compiled.php(472): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\Dev\workspace\epikureos\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
